### PR TITLE
WIP: Refactor/config

### PR DESCRIPTION
**Pull request by webmass**

### CODE REVIEW

File: components/Navbar/AppNav.tsx [hunk 1]
```diff
@@ -223,16 +220,28 @@ const AppNavConnect = () => {
 
 export const AppNav = ({ active }: { active?: string }) => {
   const [showMobileNav, setShowMobileNav] = useState(false)
-  const { activate, active: walletActive } = useWeb3React<Web3Provider>()
+  const [showWronNetModal, setShowWrongNetModal] = useState(false)
```
Comment (nourharidy) [inline components/Navbar/AppNav.tsx:223]: small typo `showWronNetModal`

File: components/Navbar/AppNav.tsx [hunk 2]
```diff
@@ -223,16 +220,28 @@ const AppNavConnect = () => {
...
     const previouslyConnected = JSON.parse(window.localStorage.getItem('previouslyConnected'))
     if (previouslyConnected) {
       activate(injectedConnector)
     }
   }
 
+  useEffect(() => {
+    if(!chainId) { return }
+    const isSupported = isSupportedNetwork(chainId);
+    setShowWrongNetModal(!isSupported);
```
Comment (nourharidy) [inline components/Navbar/AppNav.tsx:236]: wallet should be deactivated if the network is not supported

File: util/contracts.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ import {
   HARVESTER,
   LENS,
   ESCROW,
-} from '@inverse/config'
+} from '@inverse/config/constants'
 import { MulticallProvider } from '@0xsequence/multicall/dist/declarations/src/providers'
 
 export const getNewProvider = () => new InfuraProvider('homestead', process.env.INFURA_ID)
```
Comment (nourharidy) [inline util/contracts.tsx:33]: We should use Alchemy everywhere. Also remember to update .env.local example in README

**Comment by nourharidy** [comment]
We should also remove the Stake page and all associated config data. Staking is discontinued. Users who wish to unstake can do so using the https://old.inverse.finance app

**Comment by webmass** [comment]
> We should also remove the Stake page and all associated config data. Staking is discontinued. Users who wish to unstake can do so using the https://old.inverse.finance app

Alright sounds good

**Review by nourharidy — APPROVED** [review]

**Pull request merged**